lisa/tools/blobdownload.py

```diff
@@ -0,0 +1,115 @@
+import re
+from pathlib import PurePath
+from typing import Any, List
+
+from lisa.executable import Tool
+from lisa.util import LisaException
+
+from .python import Pip, Python
+
+
+class BlobDownloader(Tool):
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        self.pip = self.node.tools[Pip]
+        self.python = self.node.tools[Python]
+
+    @property
+    def command(self) -> str:
+        return "echo"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def _check_exists(self) -> bool:
+        return (
+            self.python.run(
+                '-c "from azure.storage.blob import BlobServiceClient"',
+                force_run=True,
+            ).exit_code
+            == 0
+        )
+
+    def install(self) -> bool:
+        self.pip.install_packages("azure-storage-blob")
+        return self._check_exists()
+
+    def download_blob(
+        self,
+        connection_string: str,
+        container_name: str,
+        blob_name: str,
+        file_name: str = "",
+    ) -> PurePath:
+        if file_name:
+            file_path = self.get_tool_path().joinpath(file_name)
+        else:
+            file_path = self.get_tool_path().joinpath(blob_name)
+        download_script = (
```

Comment:
This tool should be in Azure folder like other Azure tools. So it won't be implemented like this way.

Reply:
@squirrelsc could you please elaborate this `So it won't be implemented like this way.`? Thanks. 
I have tried two ways, either download the blob into the lisa run machine, then upload into the test target; or use this wrap python code, to download the blob directly into the test target.